tutoroars/plugin.py

```diff
@@ -451,6 +451,39 @@ def dbt(command: string) -> list[tuple[str, str]]:
     ]
 
 
+# Ex: "tutor local do alembic "
+@click.command(context_settings={"ignore_unknown_options": True})
+@click.option(
+    "-c",
+    "--command",
+    default="run",
+    type=click.UNPROCESSED,
+    help="""The full alembic command to run configured ClickHouse database, wrapped in
+            double quotes. The list of commands can be found in the CLI section here:
+            https://alembic.sqlalchemy.org/en/latest/cli.html#command-reference
+
+            Examples:
+
+            tutor local do alembic -c "current"
+            tutor local do alembic -c "upgrade head"
+         """,
+)
+def alembic(command: string) -> list[tuple[str, str]]:
+    """
+    Job that proxies alembic commands to a container which runs them against ClickHouse.
+    """
+    return [
+        (
+            "oars",
+            "echo 'Making alembic script executable...' && "
+            "chmod +x /app/oars/scripts/oars/alembic.sh && "
+            f"echo 'Running alembic {command}' && "
+            f"bash /app/oars/scripts/oars/alembic.sh {command} && "
+            "echo 'Done!';",
+        ),
+    ]
+
+
 # Ex: "tutor local do dbt "
 @click.command(context_settings={"ignore_unknown_options": True})
 def drop() -> list[tuple[str, str]]:
```

Comment:
I think it would make sense to do a confirmation on this? https://click.palletsprojects.com/en/8.1.x/prompts/#confirmation-prompts

Reply:
Totally agree